packages/apps/plugins/plugin-layout/src/translations.ts

```diff
@@ -20,6 +20,10 @@ export default [
         'toggle fullscreen label': 'Toggle fullscreen',
         'settings enable complementary sidebar label': 'Show context sidebar.',
         'settings show footer label': 'Show footer.',
+        'undo available label': 'Click to undo previous action.',
```

Comment:
I don’t suppose we might be able to provide the user with more information than this? I think ideally it’d say something like:

> You deleted a thread.
> Undo | Dismiss

Reply:
Can sort it out in a follow-up if we get feedback indicating folks would want this, I know it’s potentially a lot of added complexity.

Reply:
Good point, let me see if I can find a way that we could have a custom label here specified by the action.